react/src/DatePicker/DatePickerContext.tsx

```diff
@@ -257,5 +250,6 @@ const useProvideDatePicker = ({
     isDateUnavailable,
     disclosureProps,
     monthsToDisplay,
+    defaultFocusedDate: props.defaultFocusedDate,
```

Comment:
supernit: may be cleaner to destructure in the args and assign that destructured constant here for consistency instead of using assignment from `props`